### ISSUE TITLE
.deb installer installs client as `mcli` instead of `mc`

### ISSUE THREAD

**Issue by binaryfire** [minio/mc#4833]
I'm automating things with Ansible so installing from a deb makes idempotency easier.

However when installing using `https://dl.min.io/client/mc/release/linux-amd64/mc.deb` the client is installed as `mcli` (usr/local/bin/mcli) instead of `mc`. This is confusing since all the documentation refers to `mc`.

@harshavardhana [mentioned this should be fixed](https://github.com/minio/minio/discussions/18883#discussioncomment-8267487) so I've opened an issue.

**Comment by harshavardhana** [comment]
Yeah we can't install mc as mc since it conflicts so we have to install mcli.

**Comment by harshavardhana** [comment]
Locking the conversation, until it's fixed.
